external-crates/move/crates/move-compiler/src/command_line/compiler.rs

```diff
@@ -68,6 +68,8 @@ pub struct Compiler {
     vfs_root: Option<VfsPath>,
     /// Hooks to save the ASTs
     save_hooks: Vec<SaveHook>,
+    // Files to fully compile (as opposed to omitting function bodies)
+    files_to_compile: Option<BTreeSet<PathBuf>>,
```

Comment:
I do not like this. But I think removing it is part of the larger rework I described